src/Symfony/Component/Clock/CHANGELOG.md

```diff
@@ -5,3 +5,4 @@ CHANGELOG
 ---
 
  * Add the component
+ * Add `MockClock::modify()`
```

Comment:
No need to add this to the changelog, this is random compared to the previous line and all other features in the component ;)